### PR TITLE
Tag:Value reference fix. 

### PR DESCRIPTION
**Pull request by Jimmy-ahlberg**
Fixing the naming of Tag:Value and inserting a reference to the SPDX specification.

### CODE REVIEW

File: OpenChain Telco SBOM Specification.md [hunk 1]
```diff
@@ -49,16 +49,18 @@ NTIA minimum elements https://www.ntia.doc.gov/report/2021/minimum-elements-soft
 Recognizing the Telco industry need for harmonization and special requirements, possibly beyond the NTIA minimum elements, the "Telco Profile of SPDX" is proposed to ensure predictability to the industry as to the elements of an SBOM that is expected. 
 
 ### 3.5	Machine Readable Data Format
-The Telco Standard SBOM will include, at a minimum, SPDX in the following machine readable Data Format as default: Tag/Value
+The Telco Standard SBOM will include, at a minimum, SPDX in the following machine readable Data Format as default: Tag:Value
 
 #### 3.5.1	Verification material/reference material
+Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
```
Comment (CsatariGergely) [inline OpenChain Telco SBOM Specification.md:55]:
```suggestion
Tag:Value is described in section 4.4, Standard data format requirements of the SPDX 2.2 specification ([https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements](https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements))
```

File: OpenChain Telco SBOM Specification.md [hunk 2]
```diff
@@ -49,16 +49,18 @@ NTIA minimum elements https://www.ntia.doc.gov/report/2021/minimum-elements-soft
...
 #### 3.5.2	Rationale
-To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag/Value format. An entity can release additional machine readable Data Formats but they are not required to conform to the Telco SBOM specification.
+To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag:Value format. An entity can release additional machine readable Data Formats but they are not required to conform to the Telco SBOM specification.
 #### 3.6	Human Readable Data Format
-A Telco Standard SBOM will include, at a minimum, the SPDX in the following human readable Data Format as default: Tag/Value
+A Telco Standard SBOM will include, at a minimum, the SPDX in the following human readable Data Format as default: Tag:Value
 #### 3.6.1	Verification Material/reference material
+Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
```
Comment (CsatariGergely) [inline OpenChain Telco SBOM Specification.md:61]:
```suggestion
Tag:Value is described in section 4.4, Standard data format requirements of the SPDX 2.2 specification ([https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements](https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements))
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: OpenChain Telco SBOM Specification.md [hunk 3]
```diff
@@ -49,16 +49,18 @@ NTIA minimum elements https://www.ntia.doc.gov/report/2021/minimum-elements-soft
...
 
 ### 3.5	Machine Readable Data Format
-The Telco Standard SBOM will include, at a minimum, SPDX in the following machine readable Data Format as default: Tag/Value
+The Telco Standard SBOM will include, at a minimum, SPDX in the following machine readable Data Format as default: Tag:Value
 
 #### 3.5.1	Verification material/reference material
+Tag:Value is described here in SPDX 2.2 https://spdx.github.io/spdx-spec/v2.2.2/conformance/#44-standard-data-format-requirements
 #### 3.5.2	Rationale
-To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag/Value format. An entity can release additional machine readable Data Formats but they are not required to conform to the Telco SBOM specification.
+To facilitate a simplified toolchain, a machine readable version of the SBOM needs to be included. To ensure repeatability and harmonization a conformant SBOM must be in the Tag:Value format. An entity can release additional machine readable Data Formats but they are not required to conform to the Telco SBOM specification.
```
Comment (winterrocks) [inline OpenChain Telco SBOM Specification.md:57]: As above using ":" seems to be the way.

File: OpenChain Telco SBOM Specification.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vargenau) [reply OpenChain Telco SBOM Specification.md:55]: We decided that it was safer for the moment to use the ISO version of SPDX.

**Pull request merged**